keyboards/0xcb/splaytoraid/32u4/info.json

```diff
@@ -0,0 +1,24 @@
+{
+    "processor": "atmega32u4",
+    "bootloader": "qmk-dfu",
+    "build": {
+        "lto": true
+    },
+    "matrix_pins": {
+        "cols": ["F5", "F6", "F7", "F4", "B3", "B1", "B2"],
+        "rows": ["D3", "D2", "D1", "D4", "D7", "E6", "B4", "C6"]
+    },
+    "encoder": {
+        "enabled": true,
+        "rotary": [
+            {
+                "pin_a": "B5",
+                "pin_b": "B6",
+                "resolution": 4
+            }
+        ]
+    },
+    "rgblight": {
+        "pin": "D0"
+    }
```

Comment:
`RGBLIGHT` configuration shouldn't be used when the keyboard is meant to use `RGB_MATRIX` — so this should be placed in a `config.h` file.
```suggestion
```